### PR TITLE
Marketplace: Update category descriptions

### LINKED ISSUE #67171: Marketplace: Category Description (labels)
I'm creating this ticket for us to centralize, align and keep updating the Category descriptions (labels) as we get them.

**These are the ones we already have:**

Ecommerce & Business
Everything you need to turn your WordPress site into a powerful online store.

Booking & Scheduling
_Add a fully functional booking system to your site._

Analytics
_Tools to help you better understand your site's visitors and performance._

Design
_A collection of tools that will give you more control over the design of your site._

**We still need a description for these:**

Search Engine Optimization
Events Calendar
Social
Email
Security
Finance & Payments
Shipping & Delivery
Marketing
Photo & Video
CRM & Live Chat
Crowdfunding
Learning Management Systems
Widgets
Posts & Posting

@michaelpick any suggestions on these other categories? Another thing to consider is how to keep them consistent with each other.

Some initial descriptions that came from @SaxonF some time ago; https://github.com/Automattic/wp-calypso/issues/64055#issuecomment-1140626607

### PR DESCRIPTION
**Pull request by lsl**
#### Proposed Changes

* Updates category descriptions based on [this comment](https://github.com/Automattic/wp-calypso/issues/67171#issuecomment-1237452296).

#### Testing Instructions

* Cycle through each category page: http://calypso.localhost:3000/plugins/, you should see each description has been updated.

Screenshot(35)

Fixes https://github.com/Automattic/wp-calypso/issues/67171

### CODE REVIEW

File: client/my-sites/plugins/plugins-category-results-page/index.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const PluginsCategoryResultsPage = ( { category, siteSlug, sites } ) => {
 
 	const categories = useCategories();
 	const categoryName = categories[ category ]?.name || category;
-	const categoryDescription = categories[ category ]?.categoryDescription;
+	const categoryDescription = categories[ category ]?.description;
```
Comment (lsl) [inline client/my-sites/plugins/plugins-category-results-page/index.jsx:16]: All placeholder descriptions have been replaced now so should be ok to just use the original description property. (Couldn't find anything leveraging this in an unexpected way.)

**Comment by lsl** [comment]
Kind of a toss up whether it's worth waiting for translations on this one but let's just wait and deploy as part of the next sprint.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7565578

Thank you @lsl for including a screenshot in the description! This is really helpful for our translators.

**Comment by lsl** [comment]
Enough translations are back to merge this after review.

**Review by lupus2k — APPROVED** [review]
LGTM!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7565578).

**Pull request merged**